src/index.html

```diff
@@ -7,7 +8,49 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Stars</h1>
+    <div class="stars">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars">
+      <div class="stars__star stars--1"></div>
```

Comment:
do not add modifier for star's what need to be active, add it for parent element and control the active stars through scss

Reply:
done